src/elements/common/sub-header/Sort.scss

```diff
@@ -1,8 +1,10 @@
+@import '../../../styles/variables';
+
 .be-sort-selected {
     display: flex;
-    flex: 0 0 26px;
-}
+    flex: 0 0 $bdl-grid-unit * 6;
 
-.be-menu-sort.aria-menu .menu-item {
```

Comment:
`be-menu-sort` class was removed in this PR: https://github.com/box/box-ui-elements/pull/813
and no usages exist in the repo

Reply:
Do we still want to keep aria-menu?

Reply:
i don't think so since it'll apply a new styling (padding-right) that hasn't been there for several years and is likely outdated. I think this was just an obsolete declaration that should've been removed in the linked PR